### ISSUE TITLE
Improve player orientation display

### ISSUE THREAD

**Issue by Heptagon-ru** [anegostudios/VintageStory-Issues#49]
Please consider adding a numerical display of player's orientation into the coords info window, in additional to north/south/east/west indicator.
E.g. in +,+ format, four possibilities: +,+  -,-  -,+ +,-
Or adding in brackets to coords themselves, e.g.:  -1536(-),8284(+) means that if player start moving forward, first coord will decrease, second will increase.

Reason: when exploring and knowing your corrent coords and your destination coords, it is not trivial to understand where to orieng your character. Usually I have to make small test moves in different directions to see how coords start to change. Additional problem is that North is actually where negative coords are, i.e. at map's "bottom", which is not expected by US/EU/EMEA players.

Basically, since we have main info - coords - in numerical form, then direction info should also be in numerical form.

At least by a client option.

**Comment by milochristiansen** [comment]
This would be nice, but much nicer would be having the sun and moon move in the correct direction. Both together would be totally awesome.

**Comment by sirStroam** [comment]
No longer supporting suggestions here. Please move to forum suggestions.
https://www.vintagestory.at/forums/forum/5-suggestions/
